pkg/acl/acl.go

```diff
@@ -260,7 +260,7 @@ func ApplyAccessControlPolicies(ctx context.Context, operation string, httpVerb
 	var errMessage string
 
 	if err != nil {
-		errMessage = fmt.Sprintf("error in method normalization: %s", err)
+		errMessage = fmt.Sprintf("error in method normalization: %v", err)
```

Comment:
I know there are couple of `fmt.Sprintf` in our code, and I feel at some places out of those, a string concatenation would have been better probably.
Do we want to do a string concatenation here?

Reply:
Yeah, although for error handling it's probably ok since it should not be hot paths